cfme/tests/infrastructure/test_host.py

```diff
@@ -90,7 +90,7 @@ def navigate_and_select_quads(provider):
         view: the provider nodes view, quadicons already selected"""
     hosts_view = navigate_to(provider, 'ProviderNodes')
     assert hosts_view.is_displayed
-    [h.check() for h in hosts_view.entities.get_all()]
+    [h.ensure_checked() for h in hosts_view.entities.get_all()]
```

Comment:
This has been tested with cfme/tests/infrastructure/test_host.py::test_multiple_host_good_creds and the test will be added to PRT.